S/Spasm/build_tarballs.jl

```diff
@@ -0,0 +1,71 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Spasm"
+version = v"1.3.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/laurentbartholdi/spasm.git", "072719a40c837e447dfe4ae9e4941c60d9a28eda"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/spasm
+
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p1 ${f}
+done
+
+cmake -B build -DCMAKE_INSTALL_PREFIX=${prefix} -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release
+cmake --build build
+cmake --install build
+
+for t in bitmap check_cert dm echelonize kernel rank solve stack transpose vertical_swap; do
+    cp build/tools/$t $prefix/bin/
+done
+
+cp build/src/libspasm* $prefix/lib/
```

Comment:
Is this just copying one library called `libspasm` (with the appropriate extension) into the directory? If so, our standard way of doing that is the command
```
install -Dvm 755 "build/src/libspasm.${dlext}" "${libdir}/libspasm.${dlext}"
```

Reply:
OK, done. The build will fail on some architectures because it depends on a PR for FFLAS_FFPACK